GetPet/GetPet.Crawler/Parsers/Abstractions/IParser.cs

```diff
@@ -14,6 +15,9 @@ public interface IParser
 
         string ParseName(HtmlNode node);
         string ParseAgeInYear(HtmlNode node);
-        string ParseGender(HtmlNode node);
+        Gender ParseGender(HtmlNode node);
```

Comment:
you don't need to use a method for each property u parse,
just use parseItem & parse all the properties u need in that single method

Reply:
prefer to leave it this way for now as it changes frequently..